### ISSUE TITLE
prettier-path & prettierPath

### ISSUE THREAD

**Issue by joefraley** [kentcdodds/prettier-eslint-cli#9]
## TL:DR;
[Docs say `prettier-path` is an option](https://github.com/kentcdodds/prettier-eslint-cli#--prettier-path).
`prettier-eslint --help` says `prettierPath` is the option.

Both options throw an error:
```bash
$ prettier-eslint --help

Usage: prettier-eslint <globs>... [--option-1 option-1-value --option-2]

Options:
  # ...
  --prettierPath  The path to the prettier module to use
```

```bash
$ prettier-eslint index.js --prettierPath `which prettier`

Ignored unknown option: --prettierPath



$ prettier-eslint index.js --prettier-path `which prettier`

Ignored unknown option: --prettier-path
```

## Question:
I take it some combination of these should be true:
1. The README and `--help` should conform.
2. Whatever the right way is should be a known option.

What's the right option? Incidentally, `--write` actually seems to work in both cases. Just throws an error.

## To reproduce:
1. `npm i -g prettier-eslint-cli prettier` // put the tools somewhere else
2. `git clone https://github.com/joefraley/test-prettier && cd test-prettier && npm i` // get an mvp repo with an eslint config and a test file
4. `npm start` // run cli with a defined path
5. `git diff` // see that the file was changed

**Comment by kentcdodds** [comment]
Seems like a pretty significant issue. Thank you for filing this. I'll get to it when I have the time. In the meantime, if you'd like to dig deeper to see why you're getting that error, that would be really helpful. Thanks!
